adding criteria operators in approvals (#5300)
harness/developer-hub: docs/platform/approvals/custom-approvals.md

Whether the pipeline/stage stops executing depends on the stage or step [Failure Strategy](../pipelines/define-a-failure-strategy-on-stages-and-steps.md). You can specify criteria using **Conditions** and/or **JEXL Expression**. If you use them in combination, they both must evaluate to a Boolean `True` for the step to be approved.

> In **Conditions**, you can define approval criteria using outputs from the step script, [Harness expressions](../variables-and-expressions/harness-variables.md), or your custom ticketing system.  Four supported operators are `=`, `!=`, `in` and `not in`. 

Reviewer: ```suggestion
In **Conditions**, you can define approval criteria using outputs from the step script, [Harness expressions](../variables-and-expressions/harness-variables.md), or your custom ticketing system. The supported operators are `=`, `!=`, `in`, and `not in`. 
```

For example, 

> - condition for script output `field` to be in `Option1`, `Option2` or `Option3` can be specified as:

Reviewer: ```suggestion
- The condition for the script output `field` to be in `Option1`, `Option2`, or `Option3` can be specified as:
```

   ![](./static/custom-approvals-10.png)

> - condition for script output `field` to not be in either `Option1`, `Option2` or `Option3` can be specified as:

Reviewer: ```suggestion
- The condition for the script output `field` to not be in either `Option1`, `Option2`, or `Option3` can be specified as:
```

   ![](./static/custom-approvals-11.png)

:::important

> Multiple conditions with same Field are not allowed. Such usecases can be solved using `in`, `not in` operators or **JEXL Expression**.

Reviewer: ```suggestion
Multiple conditions with the same Field are not allowed. Such use cases can be solved using `in`, `not in` operators, or **JEXL Expression**.
```
